src/docfx/build/dependency/Resolve.cs

```diff
@@ -46,14 +46,15 @@ public static string GetLink(string path, object relativeTo, object resultRelati
             return link;
         }
 
-        public static XrefSpec ResolveXref(string uid, XrefMap xrefMap, Document file, DependencyMapBuilder dependencyMapBuilder, string moniker = null)
+        public static (XrefSpec spec, Document referencedFile) ResolveXref(string uid, XrefMap xrefMap, Document file, DependencyMapBuilder dependencyMapBuilder, string moniker = null)
         {
             if (xrefMap is null)
-                return null;
+                return default;
+
+            var (xrefSpec, referencedFile) = xrefMap.Resolve(uid, moniker);
+            dependencyMapBuilder.AddDependencyItem(file, referencedFile, DependencyType.UidInclusion);
```

Comment:
>AddDependencyItem [](start = 33, length = 17)

when the uid is not found, should we add it to the dependency map?

Reply:
no, we should not